.jshintignore

```diff
@@ -15,4 +15,4 @@ react/
 # Facebook Flow which are not supported by JSHint.
 modules/translation/translation.js
 
-analytics.js
+analytics.js
```

Comment:
The removal of the EOL is, obviously, a change that has not place in this PR (or any other PR for that matter).